.vib/clickhouse/goss/goss.yaml

```diff
@@ -0,0 +1,42 @@
+file:
+  /opt/bitnami/clickhouse/etc/config.xml:
+    mode: "0664"
+    filetype: file
+    exists: true
+  /opt/bitnami/clickhouse/etc/users.xml:
+    mode: "0664"
+    filetype: file
+    exists: true
+  /bitnami/clickhouse/data:
+    mode: "2750"
+    filetype: directory
+    exists: true
+http:
+  http://clickhouse:{{ .Vars.service.ports.http }}:
+    status: 200
+    allow-insecure: true
+    body:
+      - Ok
+  http://clickhouse:{{ .Vars.service.ports.http }}/replicas_status:
+    status: 200
+    allow-insecure: true
+    body:
+      - Ok
```

Comment:
Is it necessary to have both HTTP checks or would the second one suffice?

Reply:
They're testing different things. One is the instance itself and the other is the replica status